packages/ripple-nuxt-tide/modules/preview/module.js

```diff
@@ -0,0 +1,71 @@
+import { redirect } from './config/paths'
+import path from 'path'
+
+module.exports = function () {
+  const options = this.options.tide
+
+  this.options.proxy = {
+    ...this.options.proxy,
+    '/oauth/token': {
+      target: options.baseUrl,
+      onProxyReq (proxyReq, req, res) {
+        // As oauth/token is a proxied request, this authorization header
+        // for the nuxt site is not valid for the proxied target.
+        // Not removing it will present an unauthorizable user prompt.
+        proxyReq.removeHeader('authorization')
+      }
+    }
+  }
+
+  // Using `@nuxtjs/auth-next` instead of `@nuxtjs/auth` due to this bug:
+  // https://github.com/nuxt-community/auth-module/issues/761
+  // Switch to stable when v5 is released.
+  this.addModule(['@nuxtjs/auth-next', {
+    strategies: {
+      drupal: {
+        scheme: 'oauth2',
+        endpoints: {
+          authorization: `${options.baseUrl}oauth/authorize`,
```

Comment:
Hey @alan-cole we need this to support baseUrl with and without slash in the end.